test-basic/query-builder.js

```diff
@@ -2001,7 +2001,50 @@ describe('query-builder', function() {
         {'false-query':null}
         );
   });
-
+  it('should create a before-query', function(){
+   var d = new String('foo');
```

Comment:
Could the test value be something like '1562178870875' instead of 'foo'?

We don't currently check that the value is an unsigned long but might someday and, even if we don't, it's easier for someone else to understand the test if the value is valid for the data type.